client/my-sites/media-library/list.jsx

```diff
@@ -182,57 +161,114 @@ export const MediaLibraryList = React.createClass( {
 		);
 	},
 
-	renderLoadingPlaceholders: function() {
-		var itemsPerRow = this.getItemsPerRow(),
-			itemsVisible = ( this.props.media || [] ).length,
-			placeholders = itemsPerRow - ( itemsVisible % itemsPerRow );
+	onSectionRendered( { columnStartIndex, columnStopIndex, rowStartIndex, rowStopIndex } ) {
```

Comment:
Nitpick. But if we're adding new functions in the ES5 `foo: function(){}` way elsewhere in the PR, we should keep it consistent.